src/ig/IGExporter.ts

```diff
@@ -971,6 +971,19 @@ export class IGExporter {
       pathResourceDirectories = pathResources
         .map(directoryPath => path.join(this.inputPath, directoryPath))
         .filter(directoryPath => existsSync(directoryPath));
+      pathResources
+        .filter(directoryPath => directoryPath.endsWith(`${path.sep}*`))
+        .filter(directoryPath => existsSync(path.join(this.inputPath, directoryPath.slice(0, -2))))
+        .forEach(directoryPath => {
+          pathResourceDirectories.push(
+            ...readdirSync(path.join(this.inputPath, directoryPath.slice(0, -2)), {
+              withFileTypes: true,
+              recursive: true
+            })
+              .filter(file => file.isDirectory())
+              .map(dir => path.join(dir.path, dir.name))
+          );
+        });
```

Comment:
Personal preference, but I think lines 971 - 986 above might be a little cleaner as a single foreach loop. E.g.,
```
const pathResourceDirectories: string[] = [];
pathResource.forEach(pr => {
  const fullPath = path.join(this.inputPath, pr);
  // if fullPath is a directory, add to pathResourceDirectories
  // else if fullPath ends with '/*' recursively add child directories to pathResourceDirectories
});
```

Reply:
Yeah you're right. I'm not sure why I was so worried about refactoring the previous code, but I'll clean it up.

Reply:
Refactored in [6634c47](https://github.com/FHIR/sushi/pull/1427/commits/6634c47d1480dc56e2736b00349c6072307eaf9f).